backend/src/test/java/de/bund/digitalservice/ris/norms/integration/adapter/input/restapi/TimeBoundaryControllerIntegrationTest.java

```diff
@@ -269,6 +269,62 @@ void itCallsUpdateTimeBoundariesDateMalformed() throws Exception {
           .andExpect(status().isBadRequest());
     }
 
+    @Test
```

Comment:
Should we also be testing the actual response content or just the bad request status? And why did you decide to write an integration test and not a controller unit test if the goal was to test a validation happening only in the controller?